openpype/hosts/maya/api/pipeline.py

```diff
@@ -349,16 +349,8 @@ def containerise(name,
     ]
 
     for key, value in data:
-        if not value:
-            continue
-
-        if isinstance(value, (int, float)):
-            cmds.addAttr(container, longName=key, attributeType="short")
-            cmds.setAttr(container + "." + key, value)
-
-        else:
-            cmds.addAttr(container, longName=key, dataType="string")
-            cmds.setAttr(container + "." + key, value, type="string")
+        cmds.addAttr(container, longName=key, dataType="string")
+        cmds.setAttr(container + "." + key, value, type="string")
```

Comment:
Not sure if Maya convert values automatically but I think that representation id is `ObjectId`.
```suggestion
        cmds.setAttr(container + "." + key, str(value), type="string")
```

Reply:
Maya does not automatically convert all types no - however, this would change any `None` value into "None". If we're fine with that I'm happy with refactoring like this.

The ObjectId being set hasn't been an issue so far somehow. This works fine:
```python
from maya import cmds
from bson.objectid import ObjectId
id = ObjectId()
plug = "blabla_CON.representation"
cmds.setAttr(plug, id, type="string")
```

This does error:
```python
cmds.setAttr(plug, None, type="string")
```

Reply:
Shouldn't be the values validated before are really stored into the scene? Because this way you'll end up with container that has half of data anyway.

Reply:
I feel like they should be - yes. :) Haven't seen direct issues in the past either. I'll refactor this to use `str()` like you describe because many other hosts seem to do something similar.